tracer/util.go

```diff
@@ -0,0 +1,11 @@
+package tracer
+
+import opentracing "github.com/opentracing/opentracing-go"
+
+// SetError sets all three error tracking tags on the opentracing Span as
+// specified by https://github.com/opentracing/specification/blob/85d47cc/data_conventions.yaml
+func SetError(span opentracing.Span, err error) {
+	span.SetTag("error", true)
```

Comment:
FYI, you can also do `ext.Error.Set(span)` (or something like that, syntax from memory). The "message" and "event" are brand new and still need to be added to opentracing-go... I suspect that we will change opentracing-go to support an `"error.object"` tag with a Go `error` value, but – given that we haven't yet – what you're doing here is the right thing for now.